addons/core/functions/fnc_ClientInit.sqf

```diff
@@ -209,6 +204,13 @@ if (player call TFAR_fnc_isForcedCurator) then {
     GVAR(VehicleConfigCacheNamespace) setVariable ["lastRadioSettingUpdate",diag_tickTime];//Also updates Radio settings for safety
 },true] call CBA_fnc_addPlayerEventHandler;
 
+["AssignOwnerToRadios", "OnRadiosReceived", {
+    params ["_player","_radios"];
+    {
+        [_x, getPlayerUID _player] call TFAR_fnc_setRadioOwner;
+    } forEach _radios;
```

Comment:
`forEach` -> `count`

Reply:
Also that code shouldn't be needed at all. The owner is either empty or the player. If it's empty it's automatically set to the players ID. The only way for it to be someone elses ID is to add a radio with ID that already belongs to someone else via script. And as this event comes from the server that is generating the ID and taking care of it being unique. This can't happen here.
Meaning this is completly useless.

Reply:
Also as you add this EH to player it won't work with Zeus remote control anyway.